modules/ROOT/pages/custom-policies.adoc

```diff
@@ -76,8 +76,7 @@ If you receive an error similar to:
 Failed to retrieve remote metadata ${groupId}:${policyName}:1.0.0-SNAPSHOT/maven-metadata.xml: Could not transfer metadata ${orgId}:${policyName}:1.0.0-SNAPSHOT/maven-metadata.xml from/to exchange-server (https://${ANYPOINT_DNS}/exchange/maven/api/v1/organizations/${orgId}/maven): PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target -> [Help 1]
 ----
 +
-This means the certificate of the platform is not trusted by the local Maven. Similar to <<registering_servers,Registering Servers>>, the solution is to trust the certificate in the environment where Maven is running.
-For example:
+This means the certificate of the platform is not trusted by the local Maven. The solution is to trust the certificate in the environment where Maven is running. For example:
```

Comment:
```suggestion
This means the certificate of the platform is not trusted by the local instance of Apache Maven. The solution is to trust the certificate in the environment where Maven is running:
```
GLOBALLY remove the fragment "For example:" in favor of ending the previous text with a colon or making this fragment longer or more robust. See style ref.